### ISSUE TITLE
Rename repo to stex-channel

### ISSUE THREAD

**Issue by sebamarynissen** [sebamarynissen/simtropolis-channel#29]
@memo33 What do you think the name of the channel should be? I'm thinking about renaming it to stex-channel so that it becomes available on https://sebamarynissen.github.io/stex-channel/channel instead of https://sebamarynissen.github.io/simtropolis-channel/channel.

It's important to settle this now because I think I'm going to upload my everseasonal flora mod soon on the STEX and instruct users to install it via the stex channel, so we wouldn't want the url to change after that. At least not until Simtropolis has added a DNS record that makes the channel available under something like https://sc4pac.simtropolis.com.

I've also noticed that the domain name sc4pac.com is still available for $10 a year on CloudFlare. I could also pick it up and put it under https://stex.sc4pac.com, though it feels more natural to eventually put it under simtropolis.com.

**Comment by memo33** [comment]
This is not something I have a strong opinion about. I think `simtropolis-channel` sounds a bit more recognizable. Perhaps ask the admins for their opinion?

Regarding the domain, I agree it feels more natural to put it under `sc4pac.simtropolis.com` than under a custom domain.

**Comment by sebamarynissen** [comment]
Alright, I will ask them. Perhaps that they can immediately put it under the `sc4pac.simtropolis.com` subdomain, though I doubt that's something CB can do. It will probably require Dirk's intervention.

As for my mod, I think I am going to put the simtropolis channel under my `sc4pac.sebamarynissen.dev` domain name and then instruct downloaders to use that one. That way we don't reveal yet too much about what's in the works, as it just looks like it's my own channel.

**Comment by sebamarynissen** [comment]
Alright, I won't change the name of the repo - and hence the channel. It's not worth the effort and the hassle.

**Comment by memo33** [comment]
Yeah, I can understand that. At some point, I was considering to rename `sc4pac-tools` to `sc4pac` and `sc4pac` to `sc4pac-channel`, but it would have been a headache.

**Comment by sebamarynissen** [comment]
I've been playing a bit with domain names, and unfortunately GitHub pages only allows 1 custom domain name, so it's not possible to point both [sc4pac.sebamarynissen.dev](https://sc4pac.sebamarynissen.dev) and [sc4pac.simtropolis.com](https://sc4pac.simtropolis.com) to this repo. Hence I won't make this available under my own domain name as that might break things later on.

However, in the wake of this, I think I'm going to rename this repo after all, but I also would like to transfer it to an organization on GitHub. Unfortunately the Simtropolis organization is [already taken](https://github.com/simtropolis). I was hoping to use simtropolis/sc4pac-channel as organization and name for the repo, but that won't be possible. Any other ideas?

On a sidenote, the channel is now deployed under a `/channel` path, but when using custom domain names this feels a bit unneeded. I imagine it should be possible to manually remove the `/channel` by changing what folder has to be deployed to GitHub pages, right?

**Comment by sebamarynissen** [comment]
Cool, I got it to work. The channel is now available on https://sebamarynissen.github.io/simtropolis-channel or https://sc4pac.sebamarynissen.dev. I don't plan to make that last url public as per my comment above, it was just to test the domain names. To be replaced with https://sc4pac.simtropolis.com
